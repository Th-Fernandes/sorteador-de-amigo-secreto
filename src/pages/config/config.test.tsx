import { render } from "@testing-library/react"
import { RecoilRoot } from "recoil";
import { ConfigPage } from ".";

jest.mock('react-router-dom', () => {
  return {
    useNavigate: () => jest.fn()
  }
})

describe('root page behaviors', () => {
  test('it should render root page', () => {
    const { container } = render(
      <RecoilRoot>
        <ConfigPage />
      </RecoilRoot>
    );

    expect(container).toMatchSnapshot()
  })
})