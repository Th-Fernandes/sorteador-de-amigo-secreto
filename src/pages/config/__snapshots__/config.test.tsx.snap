// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root page behaviors it should render root page 1`] = `
<div>
  <header
    class="sc-dkrFOg kEqgLL"
  >
    <div
      class="header-content"
    >
      <h1>
        <img
          alt="sorteador de amigo secreto logo"
          src="/img/logo.png"
        />
      </h1>
      <img
        alt=""
        src="/img/participante.png"
      />
    </div>
  </header>
  <section
    class="sc-gswNZR gWlwQb"
  >
    <form
      class="sc-bcXHqe iHWvvt"
    >
      <div
        class="add_participants_input_container"
      >
        <label
          class="add_participant_icon"
          for="addParticipantsInput"
        >
          <span
            class="material-icons "
          >
            person_add
          </span>
        </label>
        <input
          id="addParticipantsInput"
          placeholder="Insira o nome dos participantes"
          type="text"
          value=""
        />
      </div>
      <button
        class="submit_participant_button"
        disabled=""
      >
        Adicionar
      </button>
      
    </form>
    <ul />
    <footer
      class="sc-hLBbgP hvaZoj"
    >
      <button
        class="submit-participants-btn"
        disabled=""
      >
        <img
          alt=""
          src="/img/play-logo.png"
        />
        Iniciar brincadeira!
      </button>
      <img
        alt=""
        src="/img/sacolas.png"
      />
    </footer>
  </section>
</div>
`;
